projects/test-utils/.eslintrc.json

```diff
@@ -0,0 +1,44 @@
+{
+  "extends": "../../.eslintrc.json",
+  "ignorePatterns": [
+    "!**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "parserOptions": {
+        "project": [
+          "projects/test-utils/tsconfig.lib.json",
+          "projects/test-utils/tsconfig.spec.json"
+        ],
+        "createDefaultProgram": true
+      },
+      "rules": {
+        "@angular-eslint/directive-selector": [
+          "error",
+          {
+            "type": "attribute",
+            "prefix": "lib",
```

Comment:
Let's change prefixes to `ht`

Reply:
Changed all prefix selectors rules to "ht" :tada: